Dockerfile

```diff
@@ -345,7 +345,7 @@ RUN pip install --upgrade cython && \
     pip install ktext && \
     pip install fasttext && \
     apt-get install -y libhunspell-dev && pip install hunspell && \
-    pip install annoy && \
+    pip install annoy==1.15.2 && \
```

Comment:
If you can file a bug with the context in this PR and add a comment here with a link to the bug, that would be great.

Like we did here:
https://github.com/Kaggle/docker-python/blob/master/Dockerfile#L185

From time to time, I go over these issue and see if they are resolved and if so, I remove the pin.

Thanks

Reply:
Good call, done!